packages/browser/src/index.ts

```diff
@@ -10,17 +10,19 @@ export {
     BacktraceStringAttachment,
     BacktraceUint8ArrayAttachment,
     BreadcrumbLogLevel,
+    BreadcrumbType,
```

Comment:
I use alphabetical sorting for usings/imports in VSC. Is there any reason to change that? What is your config, it looks odd vsc ordered it this way

Reply:
well, `BreadcrumbsManager` is after `BreadcrumbType` alphabetically

Reply:
wait, it's not 😄 

My VSC also sorts this automatically on save.

![image](https://github.com/backtrace-labs/backtrace-javascript/assets/9085864/cf7899fa-fc4b-47a1-babb-1ab5580cf405)

Reply:
it seems that for me it's case-sensitive, and 'T' is before 's' as a char code.
![image](https://github.com/backtrace-labs/backtrace-javascript/assets/9085864/0fb2989e-4420-40e4-84ba-a33a9ac50647)